redfish-core/lib/led.hpp

```diff
@@ -238,25 +239,56 @@ inline void
                                nlohmann::json& jsonInput)
 {
     BMCWEB_LOG_DEBUG << "Get LocationIndicatorActive";
-
     nlohmann::json& jsonIn = jsonInput["LocationIndicatorActive"];
-    dbus::utility::getAssociationEndPoints(
-        objPath + "/identifying",
-        [aResp, &jsonIn](const boost::system::error_code& ec,
-                         const dbus::utility::MapperEndPoints& endpoints) {
+
+    sdbusplus::asio::getProperty<
+        std::vector<std::tuple<std::string, std::string, std::string>>>(
+        *crow::connections::systemBus, "xyz.openbmc_project.Inventory.Manager",
+        objPath, "xyz.openbmc_project.Association.Definitions", "Associations",
+        [aResp, objPath,
+         &jsonIn](const boost::system::error_code ec,
+                  std::vector<std::tuple<std::string, std::string, std::string>>
+                      associations) {
         if (ec)
         {
-            if (ec.value() != EBADR)
-            {
-                messages::internalError(aResp->res);
-            }
+            BMCWEB_LOG_ERROR << "DBUS response error " << ec;
             return;
         }
+        if (associations.empty())
+        {
+            BMCWEB_LOG_ERROR << "Associations not found for - [ " << objPath
+                             << " ]. Skipping Get LED state.";
+            return;
+        }
+        std::string fType{get<0>(
+            associations[0])}; // TODO: check we want this always??or it varies?
```

Comment:
I think there could be more than 1  'assosications' ?
How can know which association is for LED here?